spec/.eslintrc.json

```diff
@@ -16,6 +16,10 @@
       "expectAsync": true,
       "notEqual": true,
       "it_only_db": true,
+      "it_only_mongodb_version": true,
+      "fit_only_mongodb_version": true,
+      "it_exclude_mongodb_version": true,
+      "fit_exclude_mongodb_version": true,
```

Comment:
Should we update the contributors guide?

Reply:
done